Rakefile

```diff
@@ -22,4 +22,10 @@ require 'rspec/core/rake_task'
 desc "Run specs"
 RSpec::Core::RakeTask.new(:spec)
 
+desc "Run unit and integration tests"
+task :run_all_tests do
+  ENV["ROBOT_ENVIRONMENT"] ='test'
+  exec("rspec spec integration")
```

Comment:
This is going to cause the ruby interpreter to start up (to run this), then to fork off a new ruby interpreter.  This will be slower than the existing shell script (currently in `bin/run_all_tests`)

Reply:
ok, no real reason to do this for now, and yes, nothing to do with honeybadger...I'll revise the commits in this PR to skip these changes to how the tests are run

we can revisit later if needed